sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkLauncher.java

```diff
@@ -1162,6 +1163,11 @@ private void modelResultRates(NexmarkQueryModel model) {
         return null;
       }
 
+      if (configuration.query == PORTABILITY_BATCH && options.isStreaming()) {
+        NexmarkUtils.console("Query 13 does not support streaming mode");
```

Comment:
nit - maybe have "skipping" prefix here since this reads a bit right now like it should throw an error instead of just skipping if in streaming

Reply:
Done.